tests/data/TimeoutsAndMissingTests/cdash_4.log

```diff
@@ -0,0 +1,28 @@
+[<NA> <NA>] cdash.DEBUG: simpletest@localhost {"function":"TESTING: EMAIL","build_id":"<NA>","pid":<NA>} 
+[<NA> <NA>] cdash.DEBUG: FAILED (t=2): EmailProjectExample - OSX-SIERRA-10.12.1 - Nightly {"function":"TESTING: EMAILTITLE","build_id":"<NA>","pid":<NA>} 
+[<NA> <NA>] cdash.DEBUG: A submission to CDash for the project EmailProjectExample has failing tests.
+You have been identified as one of the authors who have checked in changes that are part of this submission or you are listed in the default contact list.
+
+Details on the submission can be found at http://<NA>/<NA>/buildSummary.php?buildid=<NA>
+
+Project: EmailProjectExample
+Site: Dash20.kitware
+Build Name: OSX-SIERRA-10.12.1
+Build Time: 2009-02-23T05:04:13 EST
+Type: Nightly
+Tests failing: 2
+
+
+*Tests failing*
```

Comment:
Per our conversation with Ross & Jim, we shouldn't repeat these tests.  Only show them in the timeout section, not the failing section.